sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/AADAuthenticationFilterAutoConfiguration.java

```diff
@@ -39,7 +38,6 @@
 @ConditionalOnWebApplication
 @ConditionalOnExpression("${spring.cloud.azure.active-directory.enabled:false}")
 @ConditionalOnMissingClass({ "org.springframework.security.oauth2.server.resource.BearerTokenAuthenticationToken" })
-@ConditionalOnProperty(prefix = AADAuthenticationFilterAutoConfiguration.PROPERTY_PREFIX, value = { "credential.client-id" })
 @Import(AADPropertiesConfiguration.class)
 public class AADAuthenticationFilterAutoConfiguration {
```

Comment:
so, the AADAuthenticationFilterAutoConfiguration doesn't dependend on "client-id" or not.

Reply:
Not depend.